lib/app_gen/input_arguments/utils.ex

```diff
@@ -63,12 +63,7 @@ defmodule AppGen.InputArguments.Utils do
 
         absinthe_generator_structs
 
-      index ->
-        if is_struct(Enum.at(absinthe_generator_structs, index), AbsintheGenerator.Mutation) do
-          require IEx
-          IEx.pry()
-        end
```

Comment:
I think this whole block was temporary right?

Reply:
💯 Good catch